tests/foreman/ui/test_discoveryrule.py

```diff
@@ -121,7 +121,7 @@ def test_positive_crud_with_non_admin_user(
 
         session.discoveryrule.delete(new_rule_name)
         dr_val = session.discoveryrule.read_all()
-        assert new_rule_name not in [rule['Name'] for rule in dr_val]
+        assert new_rule_name not in dr_val
```

Comment:
@amolpati30 you need to fix/update the rest of the tests too. There are 4 more places in the code where this change is needed. For example, https://github.com/SatelliteQE/robottelo/blob/276ddd0cae7923ac6921fd9fcdb9ab5c07533190/tests/foreman/ui/test_discoveryrule.py#L156